mithril-aggregator/src/http_server/routes/root_routes.rs

```diff
@@ -123,14 +124,23 @@ mod tests {
     async fn test_root_route_ok() {
         let method = Method::GET.as_str();
         let path = "/";
-        let mut dependency_manager = initialize_dependencies().await;
-        dependency_manager
-            .signed_entity_config
-            .allowed_discriminants = BTreeSet::from([
-            SignedEntityTypeDiscriminants::MithrilStakeDistribution,
-            SignedEntityTypeDiscriminants::CardanoImmutableFilesFull,
-            SignedEntityTypeDiscriminants::CardanoStakeDistribution,
-        ]);
+        let config = Configuration {
+            signed_entity_types: Some(
+                BTreeSet::from([
+                    SignedEntityTypeDiscriminants::CardanoStakeDistribution,
+                    SignedEntityTypeDiscriminants::CardanoImmutableFilesFull,
+                    SignedEntityTypeDiscriminants::MithrilStakeDistribution,
+                ])
+                .iter()
+                .map(SignedEntityTypeDiscriminants::to_string)
+                .collect::<Vec<_>>()
+                .join(","),
```

Comment:
Is there any purpose of using this construction instead of a simpler like the following ?
```suggestion
                format!(
                    "{}, {}, {}",
                    SignedEntityTypeDiscriminants::CardanoStakeDistribution,
                    SignedEntityTypeDiscriminants::CardanoImmutableFilesFull,
                    SignedEntityTypeDiscriminants::MithrilStakeDistribution,
                ),
```

Reply:
Done ✅